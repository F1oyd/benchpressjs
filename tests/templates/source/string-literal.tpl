({""})

({"12345"})

({"  "})

({{"\\ \" "}})

({{"'sdgfhjaghjk'"}})

({{"\c\l"}})

([{{function.join, ", ", "a", "b", "c", double}}])

({{"top line\nsecond line\nthird line"}})
