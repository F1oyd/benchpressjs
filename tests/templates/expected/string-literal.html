()

(12345)

(  )

(\ " )

('sdgfhjaghjk')

(cl)

([a, b, c, true])

(top line
second line
third line)
